packages/runtime-common/index-query-engine.ts

```diff
@@ -462,6 +464,21 @@ export class IndexQueryEngine {
     return { prerenderedCards, scopedCssUrls: [...scopedCssUrls], meta };
   }
 
+  async fetchCardTypeSummary(realmURL: URL): Promise<CardTypeSummary[]> {
+    let results = (await this.query([
+      `SELECT value
+       FROM realm_meta r
+       WHERE`,
+      ...every([
+        ['r.key = ', param(RealmMetaKey.CardTypeSummary)],
+        ['r.realm_url =', param(realmURL.href)],
+      ]),
+      `ORDER BY indexed_at`,
+      `LIMIT 1`,
+    ] as Expression)) as unknown as { value: CardTypeSummary[] }[];
+    return results[0].value;
+  }
```

Comment:
it's strange that you are not also taking in a realm_version as a parameter

Reply:
Perhaps the realm_version that you should be using is the current realm version for the realm being queried. In that case you can just join on the `realm_versions` table